models/nft/nft_trades.sql

```diff
@@ -6,366 +6,57 @@
-                aggregator_name,
-                aggregator_address,
-                tx_hash,
-                block_number,
-                tx_from,
-                tx_to,
-                unique_trade_id
-        FROM {{ ref('superrare_ethereum_trades') }}
-        UNION ALL
-        SELECT
-                blockchain,
-                project,
-                version,
-                block_time,
-                token_id,
-                collection,
-                amount_usd,
-                token_standard,
-                trade_type,
-                number_of_items,
-                trade_category,
-                evt_type,
-                seller,
-                buyer,
-                amount_original,
-                amount_raw,
-                currency_symbol,
-                currency_contract,
-                nft_contract_address,
-                project_contract_address,
-                aggregator_name,
-                aggregator_address,
-                tx_hash,
-                block_number,
-                tx_from,
-                tx_to,
-                unique_trade_id
-        FROM {{ ref('zora_ethereum_trades') }}
-        UNION ALL
-        SELECT
-                blockchain,
-                project,
-                version,
-                block_time,
-                token_id,
-                collection,
-                amount_usd,
-                token_standard,
-                trade_type,
-                number_of_items,
-                trade_category,
-                evt_type,
-                seller,
-                buyer,
-                amount_original,
-                amount_raw,
-                currency_symbol,
-                currency_contract,
-                nft_contract_address,
-                project_contract_address,
-                aggregator_name,
-                aggregator_address,
-                tx_hash,
-                block_number,
-                tx_from,
-                tx_to,
-                unique_trade_id
-        FROM {{ ref('blur_ethereum_trades') }}
+FROM (
+    {% for model in nft_models %}
+    SELECT
+        blockchain,
+        project,
+        version,
+        block_time,
+        token_id,
+        collection,
+        amount_usd,
+        token_standard,
+        trade_type,
+        number_of_items, --team-query cast as decimal: CAST(number_of_items AS DECIMAL(38,0)) AS number_of_items,
+        trade_category,
+        evt_type,
+        seller,
+        buyer,
+        amount_original,
+        amount_raw, --team-query cast as decimal: CAST(amount_raw AS DECIMAL(38,0)) AS amount_raw,
+        currency_symbol,
+        currency_contract,
+        nft_contract_address,
+        project_contract_address,
+        aggregator_name,
+        aggregator_address,
+        tx_hash,
+        block_number, --**not sure where** team-query cast as decimal: CAST(number_of_items AS DECIMAL(38,0)) AS number_of_items,
```

Comment:
Not sure if these comments should stay

Reply:
we should remove those comments in a future PR, that was just my guide and forgotten 🥲